src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1284,18 +1284,25 @@ protected override void Validate(object arguments, EngineIntrinsics engineIntrin
                     null, Metadata.ValidateCountNotInArray);
             }
 
+            if (MinLength == MaxLength && len != MaxLength)
```

Comment:
What is a conclusion for new PR - leave one message and remove unneeded resource strings?